.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,124 @@
+name: 'Combine PRs'
+
+# This workflow will merge all dependabot PRs into one branch and open a new PR against dependabot-upgrades
+on:
+  workflow_dispatch:
+    inputs:
+      branchPrefix:
+        description: 'Branch prefix to find combinable PRs based on'
+        required: true
+        default: 'dependabot/'
+      combineBranchName:
+        description: 'Name of the branch to combine PRs into'
+        required: true
+        default: 'combine-prs-branch'
+
+jobs:
+  # Update dependabot-upgrades branch so that it's in sync with dev
+  merge-dev-dependabot-upgrades:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - name: Set Git config
+          run: |
+            git config user.name github-actions
+            git config user.email github-actions@github.com
+
+        - name: Merge dev to dependabot-upgrades
+          run: |
+            git fetch --unshallow
+            git pull origin
+            git checkout dependabot-upgrades
+            git merge --no-ff origin/dev --allow-unrelated-histories -m "Auto-merge main to dependabot-upgrades"
+            git push
+
+  # Get all PRs opened by dependabot
+  combine-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
+        id: fetch-branch-names
+        name: Fetch branch names
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const pulls = await github.paginate('GET /repos/:owner/:repo/pulls', {
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            });
+            branches = [];
+            prs = [];
+            base_branch = null;
+            for (const pull of pulls) {
+              const branch = pull['head']['ref'];
+              console.log('Pull for branch: ' + branch);
+              if (branch.startsWith('${{ github.event.inputs.branchPrefix }}')) {
+                console.log('Branch matched: ' + branch);
+                statusOK = true;
+                if (statusOK) {
```

Comment:
*question*: Won't a statusOK always be true here?

Reply:
Yes. This step is redundant at this point. Let me remove it